### DISCUSSION TITLE
Credit Paige Theme on the Homepage

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#52] [Q&A] [answered]
I would like to display "Paige Theme" on the homepage of my website.

I currently have a `./layouts/partials/Paige/style-first.css` that looks like:

```
#paige-authors,
#paige-breadcrumbs,
#paige-credit,
#paige-date,
#paige-keywords,
#paige-series,
.paige-authors,
.paige-date,
.paige-date-header,
.paige-series,
.paige-summary {
    display: none;
}
```

and `./content/_index.md` in which the front matter has `#paige-credit,` and looks like:

```
---
description: "Some text"
paige:
  search:
    hide_page: true
  style: |
    #paige-collections,
    #paige-sections,
    #paige-credit,
    #paige-reading-time {
        display: none;
    }
    #paige-pages {
        display: none;
    }
title: "Some text"
---
```

But "Paige Theme" is not displayed on the homepage.

**Comment by willfaught** [comment, marked as answer]
I think you need to put what you have in style-first.css in paige.style in the site params. That way it will show up in most pages, but will be overridden on the home page. When you put it in style-first.css, you end up with conflicting values for display for #paige-credit.
